Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/TabbedPage.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.iOSSpecific
+{
+	using FormsElement = Forms.TabbedPage;
+
+	public static class TabbedPage
+	{
+		private const string IsTranslucentTabBarPropertyName = "IsTranslucentTabBar";
+
+		public static readonly BindableProperty IsTranslucentTabBarProperty =
+			BindableProperty.Create(IsTranslucentTabBarPropertyName,
+				typeof(bool?), typeof(TabbedPage), default(bool?));
```

Comment:
an enum is probably better than a bool?

Reply:
> an enum is probably better than a bool?

hm. Don't think so, since this property has only two possible values (true or false)

Reply:
@StephaneDelcroix I decided to reopen this thread.

How do you see that?

```public enum TranslucentValue { Default, Translucent, NonTranslucent  }``` ? 

I lean toward to ```bool?``` just because it is much easier to use with bindings

Reply:
yes, that's what I mean. We shouldn't expose tri-state bool in any public API